packages/crayons-extended/custom-objects/src/components/form-builder/assets/form-mapper.json

```diff
@@ -494,7 +494,7 @@
         "message": "maximumLimits.maxCharsWarning"
       },
       "maxInternalNameChars": {
-        "count": 47,
+        "count": 50,
```

Comment:
Let's ensure, we validate with PM on count 50 is suffice to future use-case as well.